src/appleseed/renderer/modeling/postprocessingstage/renderstamppostprocessingstage.cpp

```diff
@@ -207,6 +229,22 @@ DictionaryArray RenderStampPostProcessingStageFactory::get_input_metadata() cons
             .insert("use", "optional")
             .insert("default", DefaultFormatString));
 
+    metadata.push_back(
+        Dictionary()
+            .insert("name", "scale_factor")
+            .insert("label", "Scale Factor")
+            .insert("type", "numeric")
+            .insert("min",
+                    Dictionary()
+                        .insert("value", "0.1")
+                        .insert("type", "soft"))
```

Comment:
By design, soft limits can be exceeded. Only hard limits (`type == "hard"`) are strictly enforced.

It would be nice if we could specify both soft and hard limits at the same time but our current system doesn't allow that yet.